changelogs/fragments/2614-influxdb_user-fix-issue-introduced-in-PR#2499.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - influxdb_user - fix bug introduced in PR #2499. Fixes https://github.com/ansible-collections/community.general/issues/2609 (https://github.com/ansible-collections/community.general/pull/2614).
```

Comment:
```suggestion
  - influxdb_user - fix bug which removed current privileges instead of appending them to existing ones (https://github.com/ansible-collections/community.general/issues/2609, https://github.com/ansible-collections/community.general/pull/2614).
```

Reply:
Fixed, thank you